src/utils/whitelistApis.ts

```diff
@@ -2767,7 +2767,7 @@ export const API_LIST = {
                 ROLE.PUBLIC,
             ],
         },
-        '/proxies/v8/operationalreports/download/:reportType/:date/:orgId/:fileName': {
+        '/proxies/v8/operationalreports/download': {
```

Comment:
Do not change the existing
Add new one with new version -> /proxies/v8/operationalreports/v2/download